AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -156,6 +156,7 @@ muzofond.fm###sveta__muzika-banner
 m.kenh14.vn##a[href="https://kenh14.vn/download-app.htm"] > img
 moviesjoy.*,himovies.to,tinyzonetv.to###gift-top
 reddit.com##.xPromoAppStoreFooter
+reddit.com##.XPromoBlockingModal
```

Comment:
@DerrickGibbs1 can you please attach a screenshot with this element? Unfortunately, I can't reproduce it.